src/main/java/org/datadog/jmxfetch/reporter/StatsdReporter.java

```diff
@@ -28,7 +28,7 @@ protected void sendMetricPoint(String metricName, double value, String[] tags) {
             this.statsDClient.stop();
             init();
         }
-        statsDClient.gauge(metricName, value, tags);
+        statsDClient.gauge(StatsdReporter.formatDataName(metricName), value, tags);
```

Comment:
`metricName` should be alias names defined in user configurations. I don't think we want/need to format it.